tools/process_symbol_map.py

```diff
@@ -93,15 +93,35 @@ class Map:
     include_list = []
     var_count = 0
 
+    # Info for error logging
+    current_file = ""
+    current_file_issues = []
+
     def parse(self, file):
-        reader = json.load(file)
-        for key in reader:
-            self.__parse(key, reader[key])
+        self.current_file = file.name
+        self.current_file_issues = []
+        
+        try:
+            reader = json.load(file)
+            for key in reader:
+                self.__parse(key, reader[key])
+
+        except json.decoder.JSONDecodeError:
+            print(f"Cannot parse '{file.name}', file does not contain valid JSON")
+            return
+        
+        # Don't print any info for succesfully parsed files
+        if len(self.current_file_issues) == 0:
```

Comment:
Would prefer the printing to be in the if statement's scope rather than an early return

Reply:
KK, will change